### PR TITLE
Calendar component available without button functionality

### PR DESCRIPTION
**Pull request by BV1BV1**
Made the calendar component available without the button functionality

## Checklist before requesting a review
- [ ] Test coverage is adequate
- [ ] Test makes sense
- [ ] Code is documented(all public methods)
- [ ] Code quality is adequate
- [ ] Feature complete
- [ ] (Almost no bugs)
- [ ] Dart analyzer completes without failure
- [ ] Changelog is updated
- [ ] Version is updated
- [ ] Readme is clean

### CODE REVIEW

File: lib/flutter_date_time_picker.dart [hunk 1]
```diff
@@ -14,3 +14,4 @@ export 'src/models/date_box_highlight_theme.dart';
 export 'src/models/date_box_selected_theme.dart';
 export 'src/models/date_time_picker_theme.dart';
 export 'src/models/date_time_picker_bar_theme.dart';
+export 'src/widgets/date_time_picker/date_time_picker.dart' show DateTimePicker;
```
Comment (FlutterJoey) [inline lib/flutter_date_time_picker.dart]: why use show in this scenario?

File: lib/src/overlay_date_time_picker.dart [hunk 2]
```diff
@@ -99,7 +103,13 @@ class _OverlayDateTimePickerState extends State<OverlayDateTimePicker> {
     debugLabel: "Overlay Date Time Picker - Button",
   );
 
-  bool _isShown = false;
+  bool _isShown = true;
+
+  @override
+  void initState() {
```
Comment (FlutterJoey) [inline lib/src/overlay_date_time_picker.dart]: no need to use initstate, you can use:

`late bool _isShown = widget.isShown;`

File: lib/src/widgets/date_time_picker/date_time_picker.dart [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  /// a [Function] that determents the icon of the button for going to the next page
+  final Widget Function(void Function()? onPressed)? onNextPageButtonBuilder;
+
+  /// a [Function] that determents the icon of the button for going to the previous page
+  final Widget Function(void Function()? onPressed)?
+      onPreviousPageButtonBuilder;
+
+  const DateTimePicker(
+      {super.key,
+      this.isShown = true,
```
Comment (FlutterJoey) [inline lib/src/widgets/date_time_picker/date_time_picker.dart]: isShown variable is not used in this widget or this widget's state. You should remove this

**Review by FlutterJoey — CHANGES_REQUESTED** [review]
Is it correct that 2 separate features have been added, as I can see these new unique use cases being created here:
- The ability to have the datepickerdialog shown from the start
- The ability to use the DateTimePicker not as an overlay, but as a widget part of the layout.

Also, the changelog and pubspece.yaml changes are missing with the increment of the version accompanied with the description of the made changes. Can you provide those?

**Comment by BV1BV1** [comment]
Only one feature added. Functionality to show dialogbox immediately in combination with a  button to show dialoogbox not deemed necessary.

**Review by FlutterJoey — APPROVED** [review]

**Pull request merged**